code/modules/mob/emote.dm

```diff
@@ -24,15 +24,20 @@
 					return target
 
 // All mobs should have custom emote, really..
-/mob/proc/custom_emote(var/m_type=1,var/message = null)
+/mob/proc/custom_emote(var/m_type=EMOTE_VISUAL,var/message = null)
 
 	if(stat || !use_me && usr == src)
 		if(usr)
 			to_chat(usr, "You are unable to emote.")
 		return
 
-	var/muzzled = istype(src.wear_mask, /obj/item/clothing/mask/muzzle)
-	if(m_type == 2 && muzzled) return
+	var/muzzled = is_muzzled()
+	if(muzzled)
+		var/obj/item/clothing/mask/muzzle/M = wear_mask
+		if(m_type == EMOTE_SOUND && M.mute & EMOTE_SOUND)
```

Comment:
err, this should check if `M.mute >= MUZZLE_MUTE_MUFFLE`, shouldn't it?

Reply:
That was how it worked before, this was a flip flop on its behavior I made.